swupd/swupd.go

```diff
@@ -66,7 +67,7 @@ const (
 	AutoUpdateWarning2 = "missing critical security patches."
 
 	// InvalidURL specifies invalid url error message
-	InvalidURL = "Invalid URL"
+	InvalidURL = "Invalid URL: Use HTTPS"
```

Comment:
@mdhorn Is it ok that we are not explicitly mentioning **FILE** as one of the allowed URI schemes for the swupd mirror?